test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -221,7 +221,7 @@ GetCounterIncrement(const std::string &counter_name)
 	std::string retval = counter_name;
 	retval += " = ";
 	retval += counter_name;
-	retval += " + 1\n";
```

Comment:
This change is not about preventing timeouts, right? Put this to a separate commit.

Reply:
It is and it is not. It's also prevents timeout since the result chunk now raises an error. It makes the state more bulletproof, as it is mentioned in the commit.

Reply:
okay, let's leave as is